core/blockchain/block_tree_error.hpp

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_BLOCK_TREE_ERROR_HPP
+#define KAGOME_BLOCK_TREE_ERROR_HPP
+
+#include <outcome/outcome.hpp>
+
+namespace kagome::blockchain {
+  enum class BlockTreeError {
```

Comment:
Some comments for this? Just to clarify to which part of project it relates to and what is the meaning of some errors for those who are not acquainted with the project well.